packages/fixie/src/voice.ts

```diff
@@ -11,10 +13,12 @@ import {
   TrackEvent,
 } from 'livekit-client';
 
-/**
- * Represents the state of a VoiceSession.
- */
+/** Represents the state of a VoiceSession. */
 export enum VoiceSessionState {
+  DISCONNECTED = 'disconnected',
+  CONNECTING = 'connecting',
+  CONNECTED = 'connected',
+  AUDIO_STARTED = 'audio_started',
```

Comment:
these states are not linear, e.g., you can be AUDIO_STARTED without being CONNECTED. The overall approach makes sense, but I think some refinement of the states is needed.

Reply:
Right - makes sense. I'll add a separate event for that.